### PR TITLE
removing the "alpha" tag from the readme

### PR DESCRIPTION
**Pull request by arschles**
**What is the problem I am trying to address?**

The readme still says the project is in alpha, but it's in beta now

**How is the fix applied?**

I updated the stuff that says we're in alpha or "the project is early" etc...

**Mention the issue number it fixes or add the details of the changes if it doesn't have a specific issue.**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,26 +15,24 @@ Welcome to the Athens project! We are a proxy server for the [Go Modules downloa
...
+There is currently an experimental public proxy, and we have plans to host a more stable public proxy with more guarantees. We also have a community of folks using Athens inside their organizations as an internal proxy. In either deployment, developers set their `GOPROXY` environment variable to point to the Athens proxy of their choice. At that point, `go get`, `go build`, and `go build`s will use the proxy to download dependencies as necessary.
 
-Athens proxies are highly configurable, so they can work for lots of different deployments. For example, public proxies can store code in cloud databases and CDNs, while internal "enterprise" deployments can use disk-based (i.e. NFS) storage.
+Athens proxies are highly configurable, so they can work for lots of different deployments. For example, proxies support a wide variety of storage drivers like cloud blob storage systems, cloud CDNs, shared disk, and even in-memory.
```
Comment (marpio) [inline README.md]: > cloud CDNs

Could you explain what do you mean by that?

File: README.md [hunk 2]
```diff
@@ -44,14 +42,14 @@ Speaking of contributing, read on!
 
 # Contributing
 
-This project is early and there's plenty of interesting and challenging work to do.
+This project has plenty of interesting and challenging work to do and a wonderful community.
```
Comment (marpio) [inline README.md]: :+1:

File: README.md (hunk 1 above)
Comment (arschles) [reply README.md]: Sure, I'll write it up in the docs

**Comment by arschles** [comment]
@marpio I changed up https://github.com/gomods/athens/pull/977#discussion_r240013186 a little bit, can you let me know what you think?

**Comment by manugupt1** [comment]
Looks like all the comments have been addressed. I am merging it. Thanks @arschles, these are amazing docs.

**Pull request merged**